searchcore/src/vespa/searchcore/proton/documentmetastore/documentmetastore.cpp

```diff
@@ -199,6 +199,36 @@ DocumentMetaStore::insert(GidToLidMapKey key, const RawDocumentMetaData &metaDat
     updateCommittedDocIdLimit();
 }
 
+namespace {
+
+// minimum dead bytes in gid to lid map before consider compaction
+constexpr size_t DEAD_BYTES_SLACK = 0x10000u;
```

Comment:
This is spread across multiple files now. I suggest that we add this to _search::CompactionStrategy_ instead.

Reply:
I suggest that is done in a separate PR
These are around in these files
```
-bash-4.2$ git grep DEAD_BYTES_SLACK
searchlib/src/tests/attribute/enum_attribute_compaction/enum_attribute_compaction_test.cpp:    constexpr size_t DEAD_BYTES_SLACK = 0x10000u;
searchlib/src/tests/attribute/enum_attribute_compaction/enum_attribute_compaction_test.cpp:    uint32_t dead_limit = DEAD_BYTES_SLACK / 8;
searchlib/src/tests/attribute/reference_attribute/reference_attribute_test.cpp:constexpr size_t DEAD_BYTES_SLACK = 0x10000u;
searchlib/src/tests/attribute/reference_attribute/reference_attribute_test.cpp:        uint64_t dropCount = DEAD_BYTES_SLACK / sizeof(Reference);
searchlib/src/vespa/searchlib/attribute/enumstore.hpp:constexpr size_t DEAD_BYTES_SLACK = 0x10000u;
searchlib/src/vespa/searchlib/attribute/enumstore.hpp:    bool compact_memory = ((dead_bytes >= DEAD_BYTES_SLACK) &&
searchlib/src/vespa/searchlib/attribute/multi_value_mapping_base.cpp:constexpr size_t DEAD_BYTES_SLACK = 0x10000u;
searchlib/src/vespa/searchlib/attribute/multi_value_mapping_base.cpp:    bool compactMemory = ((deadBytes >= DEAD_BYTES_SLACK) &&
searchlib/src/vespa/searchlib/attribute/reference_attribute.cpp:constexpr size_t DEAD_BYTES_SLACK = 0x10000u;
searchlib/src/vespa/searchlib/attribute/reference_attribute.cpp:    bool compactMemory = ((deadBytes >= DEAD_BYTES_SLACK) &&
```